### PR TITLE
Add a game menu to the map's heads-up display

### PR DESCRIPTION
**Pull request by jadtl**
This PR adds a game menu to the map's heads-up display, in the form of a drop-down button that displays a row of menu buttons right above it with a slide-in animation.

I had to put this menu on the left of the screen instead of the player's information that I moved to the bottom right of the activity because when the menu is in the right, opening it causes triggers a re-layout of the box that encloses the drop-down button and the menu buttons, causing the drop-down button to instantenously jump on the left.

I added more documentation to the HUD.

### CODE REVIEW

File: app/src/main/java/com/github/polypoly/app/map/MapActivity.kt [hunk 1]
```diff
@@ -528,7 +530,8 @@ class MapActivity : ComponentActivity() {
     }
 
     /**
-     * The HUD for the player stats
+     * The HUD for the player stats, it displays basic information such as their balance,
+     * and a button shows complete information on click
```
Comment (tamaraaPrs) [inline app/src/main/java/com/github/polypoly/app/map/MapActivity.kt:534]: you can precise the description of the param in the documentation of the function with `@param`!

File: app/src/main/java/com/github/polypoly/app/map/MapActivity.kt [hunk 2]
```diff
@@ -574,7 +577,9 @@ class MapActivity : ComponentActivity() {
     }
 
     /**
-     * The HUD that shows the stats for other players and the game
+     * The HUD that shows the stats for other players and the game, it's a button on the top left
+     * that expands and collapses a tab that contains information about the game and the other
+     * players
```
Comment (tamaraaPrs) [inline app/src/main/java/com/github/polypoly/app/map/MapActivity.kt:582]: same

File: app/src/main/java/com/github/polypoly/app/map/MapActivity.kt [hunk 3]
```diff
@@ -621,7 +628,8 @@ class MapActivity : ComponentActivity() {
     }
 
     /**
-     * The HUD for the game stats
+     * The HUD for the game stats, it displays basic information such as the current round,
+     * and a button shows complete information on click
```
Comment (tamaraaPrs) [inline app/src/main/java/com/github/polypoly/app/map/MapActivity.kt:632]: same

File: app/src/main/java/com/github/polypoly/app/map/MapActivity.kt [hunk 4]
```diff
@@ -658,7 +666,8 @@ class MapActivity : ComponentActivity() {
     }
 
     /**
-     * The HUD for the stats of other players
+     * The HUD for the stats of other players, it displays basic information such as their balance,
+     * and a button shows complete information on click
```
Comment (tamaraaPrs) [inline app/src/main/java/com/github/polypoly/app/map/MapActivity.kt:670]: same

**Review by tamaraaPrs — APPROVED** [review]
Apart from the detail of the comments, everything seems fine to me!

File: app/src/main/java/com/github/polypoly/app/map/MapActivity.kt (hunk 1 above)
Comment (jadtl) [reply app/src/main/java/com/github/polypoly/app/map/MapActivity.kt:534]: Thanks for the suggestion!

Comment (SolalPirelli) [reply app/src/main/java/com/github/polypoly/app/map/MapActivity.kt:534]: fyi, "préciser" en français -> "to specify" in English, precise is only an adjective ^^

**Pull request merged**